src/main/java/com/google/sps/servlets/NewPostServlet.java

```diff
@@ -111,8 +111,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     */
     private static String cloudStorageUpload(String fileName, InputStream fileInputStream) {
         //should edit once project id actually decided
-        String projectId = "summer22-sps-24";
-        String bucketName = "summer22-sps-24.appspot.com";
+        String projectId = "jhong-sps-summer22";
```

Comment:
Also here, please leave team project id

Reply:
Yup I changed all the project ids to team id, and put in proper settings in IAM.